apps/src/aichat/views/ChatMessageView.tsx

```diff
@@ -88,26 +88,30 @@ const ChatMessageView: React.FunctionComponent<ChatMessageViewProps> = ({
       ) : null;
   }
 
-  if (!isAssistant && !isChatHistoryView && assets && currentChannelId) {
-    footer = (
-      <div className={styles.assetRow}>
+  let header;
+  if (!isAssistant && assets && currentChannelId) {
+    header = (
+      <div className={styles.assetCol}>
         {assets.map(asset => {
           const filename = asset.filename;
-          return filename.endsWith('.pdf') ? (
-            <div key={filename} className={styles.pdfPreview}>
-              <FontAwesomeV6Icon
-                iconName="file-pdf"
-                className={styles.pdfIcon}
-              />
-              <span>{filename}</span>
-            </div>
-          ) : (
-            <img
-              key={filename}
-              alt=""
-              className={styles.imagePreview}
-              src={getAssetUrl(asset, currentChannelId, levelName)}
-            />
+          const url = getAssetUrl(asset, currentChannelId, levelName);
```

Comment:
If we end up using `FilePreview` for the image too, we could move this into the `FilePreview` component and/or remove the util entirely